tsconfig.json

```diff
@@ -13,10 +13,24 @@
     "moduleResolution": "node",
     "sourceMap": true,
     "inlineSources": true,
-    "types": []
+    "types": [
```

Comment:
You should only need to put global types here, not all of them, so it should work using just `node`.

Reply:
Also `mocha` is needed.

Reply:
Right, for global test functions 👍  Good catch.